orange/src/main/java/com/hartwig/hmftools/orange/OrangeRNAConfig.java

```diff
@@ -19,93 +28,76 @@ public interface OrangeRNAConfig {
+    static void registerConfig(final ConfigBuilder configBuilder) {
 
-        options.addOption(ISOFOX_GENE_DISTRIBUTION_CSV, true, "(Optional) Path to isofox gene distribution CSV.");
-        options.addOption(ISOFOX_ALT_SJ_COHORT_CSV, true, "(Optional) Path to isofox alt SJ cohort CSV.");
+        configBuilder.addConfigItem(RNA_SAMPLE_ID, "(Optional) The RNA sample of the tumor sample for which ORANGE will run");
 
-        options.addOption(ISOFOX_SUMMARY_CSV, true, "(Optional) Path towards the ISOFOX summary data.");
-        options.addOption(ISOFOX_GENE_DATA_CSV, true, "(Optional) Path towards the ISOFOX gene data.");
-        options.addOption(ISOFOX_FUSION_CSV, true, "(Optional) Path towards the ISOFOX fusion data.");
-        options.addOption(ISOFOX_ALT_SPLICE_JUNCTION_CSV, true, "(Optional) Path towards the ISOFOX alt splice junction data.");
+        configBuilder.addPath(ISOFOX_GENE_DISTRIBUTION_CSV, false, "(Optional) Path to isofox gene distribution CSV");
+        configBuilder.addPath(ISOFOX_ALT_SJ_COHORT_CSV, false, "(Optional) Path to isofox alt SJ cohort CSV.");
 
-        return options;
+        configBuilder.addPath(ISOFOX_DIR_CFG, false, ISOFOX_DIR_DESC);
     }
 
-    @NotNull
     String rnaSampleId();
-
-    @NotNull
     String isofoxGeneDistributionCsv();
-
-    @NotNull
     String isofoxAltSjCohortCsv();
-
-    @NotNull
     String isofoxSummaryCsv();
-
-    @NotNull
     String isofoxGeneDataCsv();
-
-    @NotNull
     String isofoxFusionCsv();
-
-    @NotNull
     String isofoxAltSpliceJunctionCsv();
 
-    @NotNull
-    static OrangeRNAConfig createConfig(@NotNull CommandLine cmd) throws ParseException {
-        if (!hasCompleteRnaConfig(cmd,
-                RNA_SAMPLE_ID,
-                ISOFOX_GENE_DISTRIBUTION_CSV,
-                ISOFOX_ALT_SJ_COHORT_CSV,
-                ISOFOX_SUMMARY_CSV,
-                ISOFOX_GENE_DATA_CSV,
-                ISOFOX_FUSION_CSV,
-                ISOFOX_ALT_SPLICE_JUNCTION_CSV)) {
-            LOGGER.debug("No proper RNA input fed to ORANGE");
+    static OrangeRNAConfig createConfig(final ConfigBuilder configBuilder) {
+
+        boolean hasRnaSampleId = configBuilder.hasValue(RNA_SAMPLE_ID);
+        boolean hasIsofoxDir = configBuilder.hasValue(ISOFOX_DIR_CFG);
+        boolean hasGeneDistribution = configBuilder.hasValue(ISOFOX_GENE_DISTRIBUTION_CSV);
+        boolean hasAltSjCohortFreq = configBuilder.hasValue(ISOFOX_ALT_SJ_COHORT_CSV);
+
+        boolean anyConfigPresent = hasRnaSampleId || hasIsofoxDir || hasGeneDistribution || hasAltSjCohortFreq;
+
+        if(!anyConfigPresent) {
+            LOGGER.info("RNA config not present");
+            return null;
+        }
+
+        boolean allConfigPresent = hasRnaSampleId && hasIsofoxDir && hasGeneDistribution && hasAltSjCohortFreq;
+
+        if(!allConfigPresent) {
+            LOGGER.warn("RNA missing required config items: rnaSampleId({}) isofoxDir({}) geneCohort({}) altSjCohort({})",
```

Comment:
Maybe this should be an ERROR/faillure? Feels like if you've specified any of these arguments you probably intend to run RNA.

Reply:
I kept with existing behaviour in Orange here.